src/components/mes-signature-pad/mes-signature-pad.scss

```diff
@@ -0,0 +1,25 @@
+mes-signature-pad {
+  button.mes-signature-pad-retry-button {
+    font-weight: bold;
+    font-size: 17px;
+    border-radius: 10px;
+    float: right;
+    position: relative;
+    top: 3.5em;
```

Comment:
Was wondering if we could use only one measuring unit, `px` or `em`?

Reply:
You make a good point. I've been trying to use `em` where possible, but in this case, I didn't think I'd want the border radius to adjust along with the font size. Probably worth a wider discussion with the team. 👍